ssp

```diff
@@ -2850,15 +2850,36 @@ sub check_for_non_default_permissions {
-        $check{'/usr/bin/httpd'} = { mode => [ '0755' ] };
-        $check{'/etc/apache2/logs/domlogs'} = { mode => [ '0755' ] };
-        $check{'/var/www/html'} = { mode => [ '0755' ] };
+        $check{'/etc/apache2'}                      = { mode => ['0755'] };
+        $check{'/etc/apache2/conf'}                 = { mode => ['0755'] };
+        $check{'/etc/apache2/conf.d'}               = { mode => ['0755'] };
+        $check{'/etc/apache2/conf.modules.d'}       = { mode => ['0755'] };
+        $check{'/etc/apache2/logs'}                 = { symlink => '/var/log/apache2', check_missing => 1 };
+        $check{'/etc/apache2/logs/error_log'}       = { mode => ['0644'], perms_help => 'If users can\t read error_log, cPanel Errors (Last 300) won\t work.' };
+        $check{'/etc/apache2/logs/domlogs'}         = { mode => ['0755'], perms_help => 'If users can\'t access logs, stats won\'t process.  See ticket 5413079.' };
+        $check{'/etc/apache2/modules'}              = { symlink => '/usr/lib64/apache2/modules', check_missing => 1 };
+        $check{'/etc/apache2/run'}                  = { symlink => '/var/run/apache2', check_missing => 1 };
+        $check{'/usr/local/apache/access_log'}      = { symlink => '/etc/apache2/logs/access_log', check_missing => 1 };
+        $check{'/usr/local/apache/bin/apachectl'}   = { symlink => '/usr/sbin/apachectl', check_missing => 1 };
+        $check{'/usr/local/apache/bin/httpd'}       = { symlink => '/usr/sbin/httpd', check_missing => 1 };
+        $check{'/usr/local/apache/bin/suexec'}      = { symlink => '/usr/sbin/suexec', check_missing => 1 };
+        $check{'/usr/local/apache/conf/httpd.conf'} = { symlink => '/etc/apache2/conf/httpd.conf', check_missing => 1 };
+        $check{'/usr/local/apache/conf/includes'}   = { symlink => '/etc/apache2/conf.d/includes', check_missing => 1 };
+        $check{'/usr/local/apache/conf/mime.types'} = { symlink => '/etc/mime.types', check_missing => 1 };
+        $check{'/usr/local/apache/conf/php.conf'}   = { symlink => '/etc/apache2/conf.d/php.conf', check_missing => 1 };
+        $check{'/usr/local/apache/domlogs'}         = { symlink => '/etc/apache2/logs/domlogs', check_missing => 1 };
+        $check{'/usr/local/apache/error_log'}       = { symlink => '/etc/apache2/logs/error_log', check_missing => 1 };
+        $check{'/usr/local/apache/htdocs'}          = { symlink => '/var/www/html', check_missing => 1 };
+        $check{'/usr/local/apache/logs'}            = { symlink => '/etc/apache2/logs', check_missing => 1 };
+        $check{'/usr/local/apache/modules'}         = { symlink => '/etc/apache2/modules', check_missing => 1 };
+        $check{'/usr/sbin/httpd'}                   = { mode => ['0755'] };
+        $check{'/var/log/apache2'}                  = { mode => ['0711'] };
+        $check{'/var/www/html'}                     = { mode => ['0755'] };
+    } else {
+        $check{'/usr/local/apache/bin/httpd'}       = { mode => ['0755'] };
```

Comment:
This is actually a symlink

root@centos6 [~]# stat /usr/local/apache/bin/httpd
  File: `/usr/local/apache/bin/httpd' -> `/usr/sbin/httpd'